client/log_client.go

```diff
@@ -189,7 +169,7 @@ func (c *LogClient) getLatestRoot(ctx context.Context, trusted *trillian.SignedL
 	}
 	// Verify root update if the tree / the latest signed log root isn't empty.
 	if resp.GetSignedLogRoot().GetTreeSize() > 0 {
-		if err := c.logVerifier.VerifyRoot(trusted, resp.GetSignedLogRoot(),
+		if err := c.LogVerifier.VerifyRoot(trusted, resp.GetSignedLogRoot(),
```

Comment:
If I've understood the point of struct embedding correctly, you no longer need to include `LogVerifier` in this expression, and `c.VerifyRoot` should do fine.

Reply:
Which you do for the `BuildLeaf` calls below.